docs/glossary.rst

```diff
@@ -0,0 +1,388 @@
+------
+
+A region in the sense used for CrateDB Cloud is a set of data centers (servers)
+grouped together on a geographic basis so as to not exceed a certain latency.
+CrateDB Cloud currently has three regions by default, but also offers the
+option to define custom regions.
+
+
+.. _glossary-saas:
+
+SaaS
+----
+
+SaaS stands for "Software-as-a-Service". It refers to a model where software is
+provided to customers on a :ref:`subscription<glossary-subscription>` basis,
+rather than a one-off payment, and is centrally hosted. CrateDB Cloud can be
+used as a service through its SaaS :ref:`offer<glossary-offer>` on |Microsoft
+Azure Marketplace|.
+
+
+.. _glossary-scale-unit:
+
+Scale unit
+----------
+
+The CrateDB Cloud :ref:`subscription plans<glossary-subscription-plan>` all
+come with a number of different scale units. Each scale unit represents an
+(additional) unit multiplying the specific combination of hardware capacity
+that applies to that plan.
+
+The relationship between scale units and :ref:`DTUs<glossary-DTU>` is subtle.
+Each scale unit added on top of the first scale unit also represents one
+*additional* DTU. However, not all plans *start* at 1 DTU. For more detailed
+information about subscription plans, scale units, and DTUs, take a look at our
+Documentation on :ref:`DTUs and Azure plans<azure-plans-dtus>`.
+
+
+.. _glossary-subscription:
+
+Subscription
+------------
+
+A subscription is - for the purposes of CrateDB Cloud - a container in which
+the CrateDB Cloud service is created and managed, supported by our cloud
+provider, Microsoft Azure. In other words, once a customer signs up for the
+CrateDB Cloud :ref:`offer<glossary-offer>` and a particular CrateDB Cloud
+:ref:`subscription plan<glossary-subscription-plan>`, they will have a
+subscription to CrateDB Cloud with Azure.
+
+The billing for that particular instance of the CrateDB Cloud service is
+managed per subscription. A given customer (say, a company) can have multiple
+subscriptions. This can be practical in case a customer wants to separate
+different instances of using the CrateDB Cloud service into different billing
+accounts.
+
+
+.. _glossary-subscription-plan:
+
+Subscription plan
+-----------------
+
+CrateDB Cloud's :ref:`offer<glossary-offer>` consists of multiple subscription
+plans. These plans are combinations of hardware specifications that are geared
+towards particular customer use cases: lower capacity vs. higher capacity, more
+storage vs. more processing power, and so forth. They can also be further
+adjusted for different :ref:`scale units<glossary-scale-unit>` per plan.
+Currently there are three subscription plans available via the |Microsoft Azure
+Marketplace|, with more to come in the near future. For more information on
+choosing the right Azure plan, refer to our Documentation `on the subject`_.
+
+
+.. _glossary-system-user:
+
+System user
+-----------
+
+In CrateDB Cloud, there are two distinct SYSTEM :ref:`users<glossary-user>`:
```

Comment:
Curious to know why System is capitalized as SYSTEM ?

And whether it should be consistent with the subtitle

Reply:
In the case of the Audit Log, the user actually appears as "SYSTEM". But I'll make this clearer.